digits/model/images/classification/views.py

```diff
@@ -475,7 +475,7 @@ def image_classification_model_top_n():
 
 def get_datasets():
     return [(j.id(), j.name()) for j in sorted(
-        [j for j in scheduler.jobs if isinstance(j, ImageClassificationDatasetJob) and (j.status.is_running() or j.status == Status.DONE)],
+        [j for j in scheduler.jobs.values() if isinstance(j, ImageClassificationDatasetJob) and (j.status.is_running() or j.status == Status.DONE)],
```

Comment:
I think you can still just use `for j in scheduler.jobs` throughout this PR. No need for `.values()`, right?